### ISSUE TITLE
Error on multi-echo data of mismatched lengths

### ISSUE THREAD

**Issue by effigies** [nipreps/fmriprep#2397]
Spent a while tracking down an issue where the number of HMC transforms mismatched the number of volumes in a BOLD series. Turned out is was multi-echo data where echos 1 & 2 had n volumes, and the third echo had n-1 volumes.

This isn't exactly our problem, but given that we calculate HMC on echo 1 and apply to the other 2, we can predict in advance that the error is going to happen and save time and pain.

See [Neurostars #18221](https://neurostars.org/t/itk-error-in-fmriprep-pipeline/18221).

**Comment by dangom** [comment]
Wouldn't this be something to add to the BIDS validator? If an fMRI multi-echo acquisition is missing echo data, then the dataset is not valid?

**Comment by tsalo** [comment]
I agree that this is more a validator issue. I've opened https://github.com/bids-standard/bids-validator/issues/1853.
